packages/react/src/components/ListBox/ListBoxSelection.js

```diff
@@ -49,7 +49,7 @@ const ListBoxSelection = ({
       onKeyDown={handleOnKeyDown}
       title={description}>
       {selectionCount}
-      <Close16 role="img" />
+      <Close16 role="img" aria-label="Clear Selection" />
```

Comment:
Should we just place the `aria-label` on the containing `div` and remove `role="img"`?